### ISSUE TITLE
Question, Are there any video to use this library?

### ISSUE THREAD

**Issue by imecar-github** [marcinbor85/cAT#42]
Hello,

I looked at the examples for usage for this library. In readme, It says this library for host devices. I have some questions.

1-For example, I have a device and i want to handle the responses. Can I use this library for handling responses?

Could you provide an example video?

**Comment by marcinbor85** [comment]
Hi.
You can use this library as long as it will be used on the host side. As
far as I know, host devices doesn't need to handling any responses. On host
side we always handle requests. As I understand You already have one device
which acts as a host. And You need to handle responses from this host
device. If so, then this library is not what You need.
MB


sob., 18 mar 2023, 09:34 użytkownik imecar-github ***@***.***>
napisał:

**Comment by imecar-github** [comment]
Yes, I want to send at commands to the device and handle the responses. Could you suggest any library or code about this?
